user-guide/modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,19 @@
 * xref:intro.adoc[]
-* xref:getting-started.adoc[]
 * xref:whats-inside.adoc[]
-* xref:reporting-issues.adoc[]
-* xref:advanced-builds.adoc[]
-* xref:faq.adoc[]
-* xref:tutorials.adoc[]
-* xref:testing-debugging.adoc[]
+* xref:getting-started.adoc[]
```

Comment:
I'm not against "getting started" before "what's inside" but that does imply another content structure because I thought what's inside could include some small examples that use the libraries.